src/d3d11/d3d11_swapchain.cpp

```diff
@@ -410,9 +438,11 @@ namespace dxvk {
           uint32_t                Repeat) {
     auto lock = pContext->LockContext();
 
-    // Bump frame ID as necessary
-    if (!Repeat)
-      m_frameId += 1;
+    if (!Repeat) {
+      m_frameId = m_presenter->lowLatencyEnabled() ?
+        m_device->getLatencyMarkers().present :
```

Comment:
Would this work with games that are not using latency markers, but only the sleep API? (e.g. Apex Legends)

Reply:
This was fixed in a previous update to this PR to only use the present latency marker if it is valid.